src/fft.jl

```diff
@@ -89,7 +89,8 @@ function FFTGrid(fft_size::Tuple{Int, Int, Int}, unit_cell_volume::T,
 
     VT = value_type(T)
     fac = 1 ./ VT.(fft_size)
-    r_vectors = map(idx -> Vec3{VT}(fac .* (idx.I .- (1, 1, 1))), CartesianIndices(fft_size))
+    r_vectors_cpu = map(idx -> Vec3{VT}(fac .* (idx.I .- (1, 1, 1))), CartesianIndices(fft_size))
```

Comment:
Add a comment that the reason here is speed. Usually we do these things, since they cannot be done easily on the GPU or since we cannot be bothered to do it on the GPU.